public_suffix_list.dat

```diff
@@ -10859,6 +10859,10 @@ cdn.prod.atlassian-dev.net
 // Submitted by Lukas Reschke <lukas@authentick.net>
 translated.page
 
+// Autocode : https://autocode.com
+// Submitted by Jacob Lee <jacob@autocode.com>
```

Comment:
Hello @jacoblee93 just a heads-up that the email address you left in the comment section is not valid, as the domain of your email address doesn't have MX records, so it won't be reachable if the PSL maintainers or volunteers try to contact you or your organization in the future. It's recommended to either fix the email account or submit an amendment to update it to a valid one. Thank you.

Reply:
Hey @groundcat, I can update but the company is dead - would it be better to remove?

Reply:
Yes, it would be better to remove it. Created #2141